src/libs/antares/study/parameters.cpp

```diff
@@ -1697,49 +1725,88 @@ void Parameters::saveToINI(IniFile& ini) const
 
     // User's playlist
     {
-        assert(yearsFilter);
-        uint effNbYears = 0;
-        bool weightEnabled = false;
-        for (uint i = 0; i != nbYears; ++i)
+        assert(not yearsFilter.empty());
+
+
+        if (version <= version800)
         {
-            if (yearsFilter[i])
+            // Playlist section must be added if at least one year is disable or one MC year weight
+            // sum is not equal to nbYears
```

Comment:
report comment with correction for MC years weight check

Reply:
I'm not sure I understand, can you clarify ?